catalogue_api/api/src/main/scala/uk/ac/wellcome/platform/api/services/ElasticsearchService.scala

```diff
@@ -5,12 +5,21 @@ import com.sksamuel.elastic4s.http.ElasticDsl._
 import com.sksamuel.elastic4s.http.HttpClient
 import com.sksamuel.elastic4s.http.get.GetResponse
 import com.sksamuel.elastic4s.http.search.SearchResponse
+import com.sksamuel.elastic4s.searches.SearchDefinition
 import com.sksamuel.elastic4s.searches.queries.BoolQueryDefinition
 import com.sksamuel.elastic4s.searches.queries.term.TermQueryDefinition
+import com.sksamuel.elastic4s.searches.sort.FieldSortDefinition
 import uk.ac.wellcome.elasticsearch.DisplayElasticConfig
 
 import scala.concurrent.Future
 
+case class ElasticsearchQueryOptions(
+  workTypeFilter: Option[String],
```

Comment:
Perhaps not here, but can we make this even more generic, with `ElasticsearchQueryOptions` having a list of `WorkFilter`, which are themselves typed?

e.g.

```scala
// Filters
sealed trait WorkFilter

case class WorkTypeFilter(workType: WorkType) extends WorkFilter
// ...
```